### PR TITLE
Use new upload layout on the upload page

### PR DESCRIPTION
**Pull request by ranzwertig**
# Definition of Done

- [ ] The code does at least what is defined in the ticket and does not affect any other functionality
- [ ] The PR has a meaningful title
- [ ] There is a link to the issue-id
- [ ] There are no 'hacks' or shortcuts refactoring is preferred
- [ ] The happy case of the app was tested at least once manually

### CODE REVIEW

File: src/app/component/upload-layout/_upload-layout.scss [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+    top: 0;
+    flex-shrink: 0;
+    z-index: z($stacking, sticky);
+  }
+
+  &__main {
+    display: flex;
+    flex-direction: column;
+    min-height: 100vh;
+    height: 100%;
```
Comment (dreampulse) [inline src/app/component/upload-layout/_upload-layout.scss]: Für was gibts hier die `height: 100%;`?

File: src/app/component/upload-layout/upload-layout.story.js [hunk 2]
```diff
@@ -0,0 +1,222 @@
...
+  deserunt mollit anim id est laborum.
+`
+
+storiesOf('UploadLayout', module)
+  .add('default', () => (
+    <UploadLayout header={navBar()} footer={footer()} stickyFooter={stickyFooter()}>
+      {lorem}
+    </UploadLayout>
+  ))
+  .add('empty', () => (
```
Comment (dreampulse) [inline src/app/component/upload-layout/upload-layout.story.js]: Eine Story für `hasModels` fehlt

File: src/app/component/upload-layout/_upload-layout.scss [hunk 3]
```diff
@@ -0,0 +1,55 @@
+.UploadLayout {
+  $stacking: content, sticky;
+
+  display: flex;
+  flex-direction: column;
+
+  &__header {
+    position: sticky;
```
Comment (webholics) [inline src/app/component/upload-layout/_upload-layout.scss]: Doesn't the parent need position relative then?

File: src/app/component/upload-layout/_upload-layout.scss [hunk 4]
```diff
@@ -0,0 +1,55 @@
...
+  }
+
+  &__stickyFooter {
+    display: none;
+    position: sticky;
+    bottom: 0;
+    background: $color-bg;
+    padding: $space-base $space-l;
+    text-align: center;
+    border-top: 2px solid $color-bg-minor;
```
Comment (webholics) [inline src/app/component/upload-layout/_upload-layout.scss]: lets use 1px for now ;)

File: src/app/container/app-layout.js [hunk 5]
```diff
@@ -1,66 +1,18 @@
...
-      />
-    }
-    footer={footer}
-  >
+import NavBarPartial from './nav-bar-partial'
+
+const AppLayout = ({children, footer}) => (
+  <PageLayout header={<NavBarPartial />} footer={footer}>
     <Container>{children}</Container>
     <Modal />
```
Comment (webholics) [inline src/app/container/app-layout.js]: modal muss hier auch noch raus

File: src/app/container/app-layout.js [hunk 6]
```diff
@@ -1,66 +1,18 @@
...
-            />
-          </>
-        }
-      />
-    }
-    footer={footer}
-  >
+import NavBarPartial from './nav-bar-partial'
+
+const AppLayout = ({children, footer}) => (
```
Comment (webholics) [inline src/app/container/app-layout.js]: sollten wir das AppLayout nicht vielleicht komplett rauswerfen? das ist jetz fast zu trivial oder?

**Pull request merged**